src/components/connect.tsx

```diff
@@ -1,8 +1,8 @@
 import { BaseWallet } from '@polkadot-onboard/core';
 import React, { useEffect, useState } from 'react';
-import { useAccount } from '../contexts/Account';
+import { useAccount, useWallets } from '../contexts';
```

Comment:
These can also be considered to be moved to contexts/hooks since it only imports the hooks this way?!

Reply:
Yeah I wasn't sure if it made sense to have everything imported from `/contexts`. I was mostly interested with having leaner imports :)

WDYT?

Reply:
I see. No strong opinion! Both works. I believe the index makes sense so they can be both imported from ./Context folder and the specific contexts. We can consider to eventually move the types to a ./types folder when things are stable so in that case types can be imported from ./types and hooks from ./contexts